okhttp-logging-interceptor/src/main/java/okhttp3/logging/HttpLoggingInterceptor.java

```diff
@@ -251,6 +259,27 @@ public Level getLevel() {
     return response;
   }
 
+  private boolean isPlainText(Buffer buffer) throws IOException {
+    Buffer out = new Buffer();
+    long bytes = buffer.size() < 64 ? buffer.size() : 64;
```

Comment:
nit: got a better name than `bytes` ? Usually we use that to refer to a `byte[]`. Maybe `byteCount` ?